src/UnscentedKalmanInversion.jl

```diff
@@ -18,6 +18,7 @@ $(TYPEDFIELDS)
         update_freq::IT = 1,
         modified_unscented_transform::Bool = true,
         prior_mean::Union{AbstractVector{FT}, Nothing} = nothing,
+        N_ens::IT = 2size(u0_mean, 1)+1
```

Comment:
I am concerned about this being too general an input (Integer). Can we instead add a keyword argument `sigma_points` that is either `symmetric` for 2p+1 or `simplex` for p+2?